src/conference/alpescraft/AlpesCraft.tsx

```diff
@@ -38,10 +48,14 @@ export const AlpesCraft: React.FC<AlpesCraftProps> = ({
 			</Sequence>
 			<Sequence name="Logo-&-title" from={30}>
 				<Title title={title} />
-				<Logo />
+				{speakers ? <Speakers speakers={speakers} /> : <Logo />}
 			</Sequence>
 			<Sequence name="Details" from={60}>
-				<Details date={date} location={location} />
+				{time ? (
+					<Details date={date} time={time} location={location} />
+				) : (
+					<Details date={date} location={location} />
+				)}
```

Comment:
Maybe one `<Details />` component with the condition in the props declaration ?

Reply:
There is but if you don't add the time information it will add an null time on the video

Reply:
Let's change Type of props of the details component to handle nullable attributes.
The condition should be in Details component not in Alpescraft

Reply:
okay